bundler/minify.js

```diff
@@ -0,0 +1,22 @@
+var minify = require('uglify-js').minify;
+
+//
+// Run uglify-js
+//
+module.exports = function (env, bundle, cb) {
+  env.log.info('minify: running through uglify-js.');
+
+  try {
+    var result = minify(bundle, { fromString: true });
```

Comment:
It looks like minify is synchronous, so the exported wrapper should probably be sync too?